src/js/components/modals/__tests__/InstallPackageModal-test.js

```diff
@@ -1,130 +0,0 @@
-jest.dontMock("../InstallPackageModal");
```

Comment:
Unit tests are obsolete and should be replaced with integration/system tests which will come in a follow up commit.